pkgs/applications/networking/cluster/mesos/default.nix

```diff
@@ -41,8 +41,14 @@ in stdenv.mkDerivation rec {
     ./nixos.patch
   ];
 
+  nativeBuildInputs = [
+    autoreconfHook
+    autoconf
+    automake
+    libtool
```

Comment:
You don't need `autoconf`, `automake` and `libtool` when including `autoreconfHook`